packager/live_packager.cc

```diff
@@ -329,7 +329,8 @@ Status LivePackager::PackageInit(const Segment& init_segment,
   packaging_params.chunking_params.segment_duration_in_seconds =
       DEFAULT_SEGMENT_DURATION;
 
-  packaging_params.mp4_output_params.include_pssh_in_stream = false;
+  packaging_params.mp4_output_params.include_pssh_in_stream =
+      config_.protection_system == ProtectionSystem::kNone ? false : true;
```

Comment:
This can be simplified to `config_.protection_system != ProtectionSystem::kNone;`